src/components/Footer/Footer.js

```diff
@@ -1,7 +1,8 @@
-import * as React from 'react';
 import PropTypes from 'prop-types';
+import * as React from 'react';
 import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
 
+import Toast from 'react-native-simple-toast';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. This doesn't affect the functionality of the code, but it's good to maintain a consistent ordering for imports for readability and maintainability. It's common to sort them alphabetically or by their type (e.g., library imports, component imports, utility imports). However, this is not a significant issue.
```diff
- import PropTypes from 'prop-types';
- import * as React from 'react';
- import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+ import * as React from 'react';
+ import PropTypes from 'prop-types';
+ import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->